viewing_party/party.py

```diff
@@ -1,23 +1,155 @@
 # ------------- WAVE 1 --------------------
 
+from curses import keyname
+from lib2to3.pgen2.tokenize import generate_tokens
+from operator import itemgetter
+from xml.sax.handler import EntityResolver
+
+from tests.test_constants import GENRE_1, MOVIE_TITLE_1, RATING_1, USER_DATA_2
+
+# adds movie data as item in movie dictionary
 def create_movie(title, genre, rating):
-    pass
+    if title == None or genre == None or rating == None:
+        return None
+    else: 
+        movie = {
+            "title" : title,
+            "genre" : genre,
+            "rating" : rating
+    }
+    return movie
+
+# adds movie to watched list
+def add_to_watched(user_data, movie):
+    user_data["watched"] = [movie]

```

Comment:
Something to consider: What will happen if `user_data["watched"]` here or `user_data["watchlist"]` on L29 contains movies already?

Instead of reassigning and overriding possibly pre-existing values here, I would look into `.append()`!